deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-daemonset-configmap.yaml

```diff
@@ -63,6 +63,15 @@ data:
             hosts:
               - 'kube-state-metrics:8080'
             period: 10s
+          - data_stream:
```

Comment:
Same here, please preserve the order.

Reply:
Oh dear. It's normally me asking for things to be correctly sorted. How embarrassing!

Fixed in 95d96e168d (the diff is awful, sorry).

Reply:
👍🏼